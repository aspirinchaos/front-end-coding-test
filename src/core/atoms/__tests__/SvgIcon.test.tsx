import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';

import { SvgIcon } from '..';

describe('SvgIcon atom', () => {
  it('renders correctly', () => {
    const { container } = render(
      <SvgIcon width={100} height={100}>
        <path />
      </SvgIcon>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
