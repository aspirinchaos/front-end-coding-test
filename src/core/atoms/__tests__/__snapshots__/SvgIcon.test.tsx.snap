// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgIcon atom renders correctly 1`] = `
<svg
  fill="currentColor"
  height="1em"
  version="1.1"
  viewBox="0 0 100 100"
  width="1em"
  xmlns="http://www.w3.org/2000/svg"
>
  <path />
</svg>
`;
